src/contexts/useArchaeologistService.ts

```diff
@@ -0,0 +1,93 @@
+      dp: '2TWTnSez81d9jqdkMCop7oQj5XEgLDXXj7MAEQbrVjo7qTZbIlcJaKB0PhD1RTxF-xxU6f2k6WM5BGPgA-rDCKEKLbEeJyEvq6TpFFa7hUvFcdq_6nP1JVuRxCywMtGqRkfJJC-69NMcEWovUrjUmZXZajoe0ZUsZAoAHcvCcXIhoRbZyNc-9TYnQldu1bQJLV5WV_B6rQVwRKFTAQamagkI5uNWYC3Czcrx4EGjGSpK7YH3zZdMp9libacU_Ux5RQB9jhSgIL94tTl68qtQYvbAjicaSLz4sPHNyuYz0-iyPitcTqt9v-RqnKkuvG79QLMeCDKNj27nu_4E4eYqSw',
+      dq: 'wOg4z_7ndHXhl0D2-TS5UE1Ygw5oowj3aWIvFlt9Fgsj4lSP6fhzU46_zjIA-Z3ccnIuka9J3tZLbaoIKDME92nV7GNna_h_-cdGegSLeqrE4t2Lz4xUSxFYiMuWdlVkpqjZysakl9qjDTpzR2q4005UrymTYsljV0tjbHH_1A1enB3MuXEvs37r7R_Hl0B9YG8lIl2ZZ7MAjuQGe4RczOqr9Cn5aUX1M5gIyikEdKX7IaLF2Nndyb6GQTPDhOXM0uLCprQ0uv6vKJOSEbMe4L5jF_PTUIGu-z6RbsXEE_ovISuJH4XshDxqX2gP-4IlfqAZ2FDxEXMC7DrXHg3KoQ',
+      qi: 'kS2AXYKfVKOdtNGiksIgs2YT4Pz-62rCeynyFlnUDc6E3eylzctLHhuWMRqWqnPwVPwnw2Fd6eUTOYZe82kVf9d0-XbeDtOOP6fS7HU30ICIvFrLTiaVEsLekUdQwoyC7s0xH99FZpdGJHzC7iNdlChybmrh4Ci5OyOLn_o8goNQBYWEXOgr6JgJLTh9BXvDEbNLLJ0JTZFQeLVqQeimPDSoAwBFCLBBwOd6CFOr5wTc9Pkt8cDrpcf9usU-61uu3Lr4-3PG4yUUQyphnDy84DgpUyuF5JNy0KgtuZaMMuAZJTb_gKN3IdLUHV9phqwrSKylLn-enGJ0f4FTpfcDEQ',
+    };
+
+    const address = await arweave.wallets.getAddress(key);
+    console.log(
+      'start upload for sarcophagus:',
+      address,
+      sarcoId,
+      await arweave.wallets.getBalance(address)
+    );
+    const tx = await arweave.createTransaction({ data: file }, key);
+    tx.addTag('Content-Type', 'plain/text');
+    await arweave.transactions.sign(tx, key);
+
+    let uploader = await arweave.transactions.getUploader(tx);
+
+    while (!uploader.isComplete) {
+      await uploader.uploadChunk();
+      console.log(
+        `${uploader.pctComplete}% complete, ${uploader.uploadedChunks}/${uploader.totalChunks}`
+      );
+    }
+    return tx.id;
+  };
+
+  const updateStatus = async () => {
+    const arweave = initArweave();
+    const currentTextArweaveAddress = 'Xm17-cZJjcx-jc_UL5me1o5nfqC2T1mF-yu03gmKeK4';
+    const txId = await arweave.wallets.getLastTransactionID(currentTextArweaveAddress);
+    console.log('txId:' + txId);
+
+    const res = await arweave.transactions.getStatus(txId);
+    if (res && res.status === 200) {
+      if ((res.confirmed as unknown as string) === 'Pending') {
+        setStatus({ status: 'Pending', confirmations: 0 });
+      } else {
+        setStatus({
+          status: 'Success',
+          confirmations: res.confirmed?.number_of_confirmations || 0,
+        });
+      }
+    }
+    console.log(res);
+  };
+
+  const getConfirmations = async (txId: string | undefined): Promise<number> => {
+    if (!txId) return 0;
+    const arweave = initArweave();
+    const res = await arweave.transactions.getStatus(txId);
+    if (res && res.status === 200) {
+      if ((res.confirmed as unknown as string) === 'Pending') {
+        return 0;
+      } else {
+        return res.confirmed?.number_of_confirmations || 0;
+      }
+    }
+    return 0;
+  };
```

Comment:
Do we need to know the # of confirmations on the TX? At what point does `await arweave.transactions.getStatus(txId)` return `200` where the status isn't `Pending` (is it immediately after the first confirmation)?

Guess I'm wondering if we could trust that a non-pending `200` TX from arweave.net is enough info for us to trust the TX is complete (and not need to fetch the confirmations number).